content/academy/puppeteer_playwright/proxies.md

```diff
@@ -0,0 +1,180 @@
+---
+title: VI - Using proxies
+description: Understand how to use proxies in your Puppeteer and Playwright requests, as well as a couple of the most common use cases for proxies.
+menuWeight: 7.6
+paths:
+    - puppeteer-playwright/proxies
+---
+
+# [](#using-proxies) Using proxies
+
+[Proxies]({{@link anti_scraping/proxies.md}}) are a great way of appearing as if you are making requests from a different location. A common use case for proxies is to avoid [geolocation]({{@link concepts/geolocation.md}}) restrictions. For example your favorite TV show might not be available on Netflix in your country, but it might be available for Vietnamese Netflix watchers.
+
+In this lesson, we'll be learning how to use proxies with Playwright and Puppeteer. This will be demonstrated with a Vietnamese proxy that we got by running [this](https://apify.com/mstephen190/proxy-scraper) proxy-scraping actor on the Apify platform.
+
+## [](#adding-a-proxy) Adding a proxy
+
+First, let's go ahead and create a variable called `proxy` which will point to our proxy server:
+
+> Note that this proxy may no longer be working at the time of reading. If you don't have a proxy to use during this lesson, we recommend using Proxy Scraper for a list of free ones, or checking out [Apify proxy](https://apify.com/proxy)
+
+```marked-tabs
+<marked-tab header="Playwright" lang="javascript">
+import { chromium } from 'playwright';
+
+// our proxy server
+const proxy = '103.214.9.13:3128';
```

Comment:
I can't decide if this is confusing or not. We define the variable, but we don't use it anywhere in this example. Maybe we should show only the one line with the variable definition as the code example?

Reply:
I am showing them the boilerplate code for the rest of the lesson + the variable declaration at the same time